app/javascript/components/lesson-preview/index.jsx

```diff
@@ -22,10 +25,30 @@ const LessonPreview = () => {
     }
   };
 
+  const handleClick = () => {
+    const encodedContent = encodeContent(content);
+    const link = generateLink(encodedContent);
+    navigator.clipboard.writeText(link).then(() => setCopied(true));
+  };
+
   useEffect(() => {
     Prism.highlightAll();
   }, [convertedContent]);
 
+  useEffect(() => {
```

Comment:
@KevinMulhern Do we need to also test the component itself?

Reply:
Yeah good shout, we can add unit tests around the button text and copying to clip board behaviour.

Reply:
I'll go ahead and figure that out then :)